src/utils/Processing.ts

```diff
@@ -240,6 +241,26 @@ export function fillTank(rawFSHes: RawFSH[], config: Configuration): FSHTank {
   return new FSHTank(docs, config);
 }
 
+function checkNullValuesOnArray(resource: any): void {
+  const resourceType = resource instanceof InstanceDefinition ? 'Instance' : resource.resourceType;
+  const resourceName =
+    resource instanceof InstanceDefinition ? resource._instanceMeta.name : resource.title;
```

Comment:
Instead of `resource.title` I would actually use `resource.name ?? resource.id`.  We rarely use `title` to identify a resource.